### ISSUE TITLE
About the detail of Prompt design

### ISSUE THREAD

**Issue by MeL0D** [hemingkx/TokenSkip#4]
Excellent work, I'm very interested in this.

I would like to ask how the prompt are designed during training and inference. I tried to use llama3.1-8B-Instruct on the gsm8k dataset to fine-tune it to train its chain of thought. But the accuracy result I got is only 73 which is much lower than the meta official 84.5. I noticed that the result of llama3.1 using raw cot as mentioned in your paper is 86.2. I wonder if there is something wrong with my prompt words design? I would like to ask you how your prompt words are designed?

This is the training sample I designed.
![Image](https://github.com/user-attachments/assets/ebadf5c8-78fb-44ae-a910-8a1ccfa2336f)

**Comment by hemingkx** [comment]
Thx for your interest in our work!

We tried to use CoT ground truths from GSM8K to train LLaMA and observed similar performance degradation. We suspect the reason is that LLaMA-3.1 already performs quite well on this task, and the CoT ground truths of GSM8K misalign with the CoTs of LLaMA. To maintain original performance, as illustrated in the paper and our readme, we first obtain the original CoTs from LLaMA-3.1 (rather than using ground truths of GSM8K) and conduct compression on these CoTs.

Heming

**Comment by AlbertTan404** [comment]
> Thank you for your answer. I retried the original llama3.1 based on your suggestion and found that the results are indeed closer to the values in the paper. I thank you from the bottom of my heart for the help you have provided.

Kindly asking which few-shot prompt are you using :)
